src/Run.mss

```diff
@@ -45,13 +45,18 @@ function DoExport (filename) {
     Self._property:libmei = libmei4;
     libmei.destroy();
 
-    // Deal with the Progress GUI
     // set the active score here so we can refer to it throughout the plugin
     Self._property:ActiveScore = Sibelius.ActiveScore;
+    if (Self._property:ActiveScore = null)
+    {
+        Sibelius.MessageBox('No score to convert. Can not export to ' & filename);
```

Comment:
"Cannot"

Reply:
It might help to say something more specific, though. Like "Could not find an active score. Unable to export to " & filename.

Reply:
@ahankinson Thanks for checking – I followed your suggestions.